### PR TITLE
Post: 2020 Interview Retro

### PR DESCRIPTION
**Pull request by agude**
Why write a data post when I can ramble for 2 pages?!

### CODE REVIEW

File: _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+decide quickly have a strong advantage in recruiting talent.
+
+### Salary Negotiation
+
+The first conversation with every recruiter in 2017 involved talking about
+present salary and future expectations. In 2020, not a single recruiter asked
+about compensation until after they had made a verbal offer. This surprised me
+because waiting to open negotiations until that late in the process offers the
+candidate an advantage; the company has already spent considerable resources
+in interviewing!
```
Comment (Veldrina) [inline _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md:58]:
```suggestion
candidate an advantage as they now know that the company has strong interest in them.
```

File: _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+binary search tree][tweet] or some other question that had never come up in my
+job as a data scientist. I was able to solve these problems when I had seen
+them before in my studies or could come up with the _"trick"_ on the fly. I
+passed about half of my technical screens.
+
+[tweet]: https://twitter.com/mxcl/status/608682016205344768
+
+In 2020, my experience was vastly different. Only one screen was even close to
+"invert this BST", and that was a mistake where they later admitted that I was
+given the _software engineering_ technical hone screen accidentally.
```
Comment (Veldrina) [inline _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md:85]:
```suggestion
accidentally given the _software engineering_ technical phone screen.
```

File: _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+All of the other screens involved reasonable questions that would come up in a
+data scientist's day to day work like manipulating a dataset, calculating some
```
Comment (Veldrina) [inline _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md:88]:
```suggestion
data scientist's daily work, like manipulating a dataset, calculating some
```

File: _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+simple features, or implementing really simple algorithms or metrics. With
+these more applied questions, I passed all of my technical screens!
+
+### Virtual On-Sites
+
+In 2017 on-sites were on site! In 2020 they are done via video conferencing. I
+thought virtual on-sites would be less draining, but I felt even more
```
Comment (Veldrina) [inline _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md:95]:
```suggestion
thought virtual on-sites would be less draining, but I actually felt even more
```

File: _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+exhausted after them. I missed the opportunity to get lunch with a team member
+that the normal on-site provided, but I still felt I was able to connect well
+with the interviewers.
```
Comment (Veldrina) [inline _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md:98]:
```suggestion
exhausted after them. However,  I still felt I was able to connect well
with the interviewers despite no longer having the opportunity to eat lunch 
with a team member as in normal on-sites.
```

File: _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+
+I actually found myself looking forward to the coding challenges because, with
+the improvement of coding on **an actual computer**, they were a nice break
+from the other interviews. Just like the technical screens these questions
+were all directly applicable to the work I would be doing.
+
+#### Open-ended Problems and Behavioral Questions
+
+This time there were more open-ended interviews that dug into some problem the
+business had (like "How would you help us filter spam?") or went really deep
```
Comment (Veldrina) [inline _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md:114]:
```suggestion
business had (e.g. "How would you help us filter spam?") or went really deep
```

File: _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+on a project I had worked on previously. I would get some of these questions
```
Comment (Veldrina) [inline _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md:115]:
```suggestion
exploring a project I had worked on previously. Although I was asked some of these questions
```

File: _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+during my previous years of interviewing, but they felt much more effective in
```
Comment (Veldrina) [inline _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md:116]:
```suggestion
during my previous years of interviewing they felt much more effective in
```

File: _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+|------------------|----------:|-----------------:|------------------------:|-------------:|
+| **DocuSign**     |  {{pass}} |         {{pass}} | {{declined}}[^docusign] |          --- |
+| **Grand Rounds** |  {{pass}} |         {{pass}} |                {{fail}} |          --- |
+| **Salesforce**   |  {{fail}} |              --- |                     --- |          --- |
+| **Square**       |  {{pass}} |         {{pass}} |                {{pass}} | {{accepted}} |
+| **Stripe**       |  {{pass}} |         {{pass}} |                {{fail}} |          --- |
+| **Twitch**       |  {{pass}} |         {{pass}} |                {{pass}} | {{declined}} |
+| **Twitter**      |  {{pass}} |         {{pass}} |                {{fail}} |          --- |
+
+I am very happy with how well technical screens went this time around, as
```
Comment (Veldrina) [inline _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md:150]:
```suggestion
I am very happy with how well the technical screens went this time around, as
```

File: _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md [hunk 10]
```diff
@@ -0,0 +1,175 @@
...
+mentioned above. I also feel good about the on-site to offer rate. I felt that
+the Twitch, Twitter, and Square interviews all went well, so I am disappointed
+by the lack of an offer from Twitter.
+
+The Stripe interview revealed the position to be less well aligned with my
+skill set[^ab] and career aspirations than I'd hope, so I think the reject
+there is fair.
```
Comment (Veldrina) [inline _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md:157]:
```suggestion
skill set[^ab] and career aspirations than I'd hoped, so I think their decision
to not extend an offer was fair.
```

File: _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md [hunk 11]
```diff
@@ -0,0 +1,175 @@
...
+
+I generally do not apply to work for startups, for a variety of reasons
```
Comment (Veldrina) [inline _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md:159]:
```suggestion
I generally do not apply to work for startups, for a variety of reasons both
```

File: _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md [hunk 12]
```diff
@@ -0,0 +1,175 @@
...
+personal and [financial][sense]. I agreed to interview at Grand Rounds after a
```
Comment (Veldrina) [inline _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md:160]:
```suggestion
personal and [financial][sense]. However, I agreed to interview at Grand Rounds after a
```

**Review by Veldrina — CHANGES_REQUESTED** [review]

File: _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md [hunk 13]
```diff
@@ -0,0 +1,175 @@
...
+image: /files/2020-interviewing/nighhawks_crop.jpg
+image_alt: >
+  A crop of Hopper's Nighthawks, showing a man in a suit sitting alone at the
+  counter of a diner.
+categories: python_patterns
+---
+
+{% include lead_image.html %}
+
+I started looking for a job this year, in the [middle of a pandemic][covid],
```
Comment (charles-uno) [inline _drafts/2020-09-23-interviewing_for_data_science_positions_in_2020.md:16]: Not sure you need to mention COVID twice in the same sentence

**Pull request merged**